examples/nats-pub/main.go

```diff
@@ -38,8 +38,11 @@ func showUsageAndExit(exitcode int) {
 func main() {
 	var urls = flag.String("s", nats.DefaultURL, "The nats server URLs (separated by comma)")
 	var userCreds = flag.String("creds", "", "User Credentials File")
-	var showHelp = flag.Bool("h", false, "Show help message")
+	var tlsClientCert = flag.String("tlscert", "", "TLS client certificate file")
+	var tlsClientKey = flag.String("tlskey", "", "Private key file for client certificate")
+	var tlsRootCA = flag.String("tlsrootca", "", "Root CA file for server verification")
```

Comment:
How about this one being `cacert` so that it is more like `curl`?

```
     --cacert <file> CA certificate to verify peer against
```

Reply:
Fine with me. Was trying to be as close to the existing naming conventions as possible. A client user would potentially stumble upon this doc while setting up tls: https://docs.nats.io/developing-with-nats/security/tls
There it's called RootCA, just like the function handling the cert

Reply:
Thanks for the link, I think you are right it should be like the options from the server where the options are:

```
> nats-server --tls --tlscert=server-cert.pem --tlskey=server-key.pem --tlscacert rootCA.pem --tlsverify
```

So using `--tlscacert` would be bettter in this case I think.

Reply:
Aligned the naming to the nats-server flags.
tlsrootca is now called tlscacert
The two nats-server flags "--tls" & "--tlsverify" are rather server specific and are therefore omitted.